diff/diff.go

```diff
@@ -101,6 +105,10 @@ func contentSearch(report *Report, possiblyRemoved []string, oldIndex map[string
 				redactSecrets(oldContent, newContent)
 			}
 
+			if len(options.SuppressedAnnotations) > 0 || len(options.SuppressedLabels) > 0 {
+				suppressAnnotationsAndLabels(oldContent, newContent, options)
```

Comment:
Presuming all `diffMappingResults` calls need to be preceded by the new function calls, can we instead put this code inside `diffMappingResults` for maintainability, changing the third parameter of it to the `options` instead of a bool? 🐱

Reply:
Sorry for the late answer, I have running code also handling nested labels/annotations in pod templates, but I think it quickly gets too specialized, having to handle all cases one by one (CronJob, Job, Deployment, StatefulSet, etc.)

I think having a render script hook before diff could make sense, that would make it possible to run a custom script deleting/sanitizing required fields. 

I know there are post-renderer flags passed to helm, but these are only applied to the target helm template command.

WDYT?